### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by andreaq**
Replace this sentence with a brief description of the awesome changes your PR has. 😊

---

## ✅️ By submitting this PR, I have verified the following

- [ ] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [ ] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [ ] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

**Comment by andreaq** [comment]
Avoid Colorblindness Issues by Using Sufficient Contrast

File: src/pages/certifications/responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast/index.md [hunk 1]
```diff
@@ -3,8 +3,21 @@ title: Avoid Colorblindness Issues by Using Sufficient Contrast
 ---
 ## Avoid Colorblindness Issues by Using Sufficient Contrast
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+<!-- This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/responsive-web-design/applied-accessibility/improve-readability-with-high-contrast-text/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.-->
```
Comment (Asjas) [inline src/pages/certifications/responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast/index.md]: Please remove this comment.

File: src/pages/certifications/responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast/index.md [hunk 2]
```diff
@@ -3,8 +3,21 @@ title: Avoid Colorblindness Issues by Using Sufficient Contrast
...
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+<!--<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.-->
```
Comment (Asjas) [inline src/pages/certifications/responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast/index.md]: Also remove this comment as well please.

**Comment by Asjas** [comment]
Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :tada:

**Pull request merged**